Gemfile.lock

```diff
@@ -156,5 +156,8 @@ DEPENDENCIES
   syntax_tree (>= 6.1.1, < 7)
   tapioca (~> 0.13)
 
+RUBY VERSION
+   ruby 3.3.4p94
```

Comment:
🤔 will this be noisy if people have different patch levels of Ruby on their machines?

Reply:
It's the Ruby version bundler will prefer using. The `ruby` declaration in `Gemfile` though is a requirement.